src/common/utils/api-id.config.ts

```
@@ -1,41 +1,60 @@
+  ACADEMICYEAR_GET: "api.academicyear.get",
+  FORM_GET: "api.form.read",
+  FORM_CREATE: "api.form.create",
+  ADD_COHORT_TO_ACADEMIC_YEAR: "api.create.cohortAcademicYear",
+  TENANT_CREATE: "api.tenant.create",
+  TENANT_UPDATE: "api.tenant.update",
+  TENANT_DELETE: "api.tenant.delete",
+  TENANT_SEARCH: "api.tenant.search",
+  TENANT_LIST: "api.tenant.list",
+  SEND_OTP: "api.send.OTP",
+  VERIFY_OTP: "api.verify.OTP",
+  SEND_RESET_OTP: 'api.send.reset.otp'
```

ESLint

[error] Replace `'api.send.reset.otp'` with `"api.send.reset.otp",`

(prettier/prettier)